src/app/shared/url-config.ts

```diff
@@ -0,0 +1,17 @@
+export const sourceUrl = [
```

Comment:
This can be confusing in a longer run. Please prefer using object with proper keys for this purpose - 
```js
export default {
  loklak: {
    api: 'https://api.loklak.org',
    blog: 'http://blog.loklak.net',
    ...
  },
  github: {
    ...
  },
  ...
}; 
```